db/schema.rb

```diff
@@ -46,10 +48,10 @@
     t.boolean "has_student_loan"
     t.integer "student_loan_country"
     t.integer "student_loan_courses"
-    t.integer "student_loan_start_date"
-    t.integer "student_loan_plan"
     t.integer "payroll_gender"
     t.text "verified_fields", default: [], array: true
+    t.integer "student_loan_start_date"
+    t.integer "student_loan_plan"
```

Comment:
Did the order of these mean to change in this commit?

Reply:
Gaaaah. This keeps happening. I've tried dropping and recreating the DB from scratch and it still happens on every migration 😠